service/src/main/resources/HTMLTemplate.html

```diff
@@ -458,291 +458,291 @@ <h3 class="heading heading--two">Attending the hearing</h3>
     </div>
   </dl>
 
-  <h3 class="heading heading--two">Child
-    {% if case_data.children is not empty and
-    case_data.children.additionalChildren is not empty %}
-      1
-    {% endif %}
-  </h3>
-  <dl class="data-list">
-    <div class="data-list__row">
-      <dt class="data-list__title">Name</dt>
-      <dd class="data-list__definition">
-        {% if case_data.children is not empty and
-        case_data.children.firstChild is not empty and
-        case_data.children.firstChild.childName is not empty %}
-          {{ case_data.children.firstChild.childName }}
-        {% else %}
-          -
-        {% endif %}
-      </dd>
-    </div>
-    <div class="data-list__row">
-      <dt class="data-list__title">Age</dt>
-      <dd class="data-list__definition">
-        {% if case_data.children is not empty and
-        case_data.children.firstChild is not empty and
-        case_data.children.firstChild.childDOB is not empty %}
-          {{ case_data.children.firstChild.childDOB | age }}
-        {% else %}
-         -
-        {% endif %}
-      </dd>
-    </div>
-    <div class="data-list__row">
-      <dt class="data-list__title">Date of birth</dt>
-      <dd class="data-list__definition">
-        {% if case_data.children is not empty and
-        case_data.children.firstChild is not empty and
-        case_data.children.firstChild.childDOB is not empty %}
-          {{ case_data.children.firstChild.childDOB | date("d MMMM yyyy", existingFormat="yyyy-MM-dd") }}
-        {% else %}
-          -
-        {% endif %}
-      </dd>
-    </div>
-    <div class="data-list__row">
-      <dt class="data-list__title">Gender</dt>
-      <dd class="data-list__definition">
-        {% if case_data.children is not empty and
-        case_data.children.firstChild is not empty and
-        case_data.children.firstChild.childGender is not empty %}
-          {% if case_data.children.firstChild.childGender == 'They identify in another way'
-          and case_data.children.firstChild.childGenderIdentification is not empty %}
-            <span class="u-block">{{ case_data.children.firstChild.childGenderIdentification }}</span>
+  {% if case_data.children is not empty and case_data.children1 is empty %}
```

Comment:
I will review this HTML template another time. I am not prepared for 793 line changes

Reply:
yeah sorry, I still haven't got this in me... haha